packages/reference/src/assets/WrappedAssetCard/WrappedAssetLink.tsx

```diff
@@ -58,6 +66,10 @@ export const WrappedAssetLink = (props: WrappedAssetLinkProps) => {
           entityType="Asset"
           entityId={props.asset.sys.id}
           status={status}
+          useLocalizedEntityStatus={props.useLocalizedEntityStatus}
+          entity={props.asset}
```

Comment:
Seems redundant to pass the entity and the entityId here. Maybe remove the entityId as a prop?

Reply:
Ah good catch, will remove it 👍